distributor-api-v1/use-cases/on-session-payments.md

```diff
@@ -0,0 +1,30 @@
+# On session payments
+
+Whenever a user create reservation there is an option to allow him pay directly. This needs to be correctly configured via
+settlement rules on `RateGroup`. We provide this functionality only for `Rates` where the payment should be automatically
+taken immediately when the reservation group is created. In order to simplify the workflow we offer this functionality
```

Comment:
The configuration is more complex. We are charging only rates that are settled right now and the method is set to charge credit card.